code/swb_base/ISWBPlayer.cs

```diff
@@ -34,13 +34,9 @@ public interface ISWBPlayer : IEntity
     public void ScreenShake(ScreenShakeStruct screenShake);
 
     /// <summary>
-    /// Whether the player is model editing
+    /// Whether the player is looking at their gun (as if model or attachment editting.)
     /// </summary>
-    public bool IsModelEditing();
-    /// <summary>
-    /// Whether the player is editing attachments
-    /// </summary>
-    public bool IsAttachmentEditing();
+    public bool IsLookingAtGun { get; }
```

Comment:
`IsLookingAtGun` could be confused with inspecting, I'd suggest renaming this to `IsLookingAtGun`

Reply:
Renamed